platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -298,6 +289,33 @@ MigrationZipFile newZipFileFor(Path exportDirectory) {
     }
   }
 
+  @VisibleForTesting
+  Properties loadMigrationProperties() {
+    try {
+      return AccessUtils.doPrivileged(
+          () -> {
+            try (InputStream inputStream =
+                new FileInputStream(
+                    Paths.get(
+                            System.getProperty(ConfigurationMigrationManager.DDF_HOME_KEY),
+                            "etc",
+                            ConfigurationMigrationManager.MIGRATION_PROPERTIES_FILENAME)
+                        .toString())) {
+              Properties props = new Properties();
+              props.load(inputStream);
+              return props;
+            }
+          });
+    } catch (IOException e) {
+      LOGGER.error(
+          String.format(
+              "unable to load migration properties from '%s'; ",
+              ConfigurationMigrationManager.MIGRATION_PROPERTIES_FILENAME),
+          e);
+      throw new IOError(e);
```

Comment:
Why is that? Honestly, I was trying to be consistent with the constructor of this class: https://github.com/codice/ddf/blob/ccbf9477814353888515b954d9c0a700a552abfd/platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java#L121
How is this case different?

Reply:
Actually, now that I look at it, and because I coded the constructor 😆 , I thin `IOError` is better since we are trying to stop the whole thing if the file is not available. Kind of like an internal error.